src/demo/ui.jsx

```diff
@@ -471,8 +466,7 @@ const styles = {
     }
   },
   infoIcon: {
-    width: 28,
-    height: 28
+    padding: '5px 12px'
```

Comment:
On small responsive sizes this seems to turn the info icon into a non-circle...

Reply:
yeah, i'm really struggling with this one. i tried applying percentage padding to it, but it wasn't working as expected